integration-tests/src/test/java/io/unitycatalog/integrationtests/BaseSparkTest.java

```diff
@@ -1,85 +1,90 @@
 package io.unitycatalog.integrationtests;
 
-import lombok.Getter;
-import lombok.SneakyThrows;
-import org.apache.spark.sql.SparkSession;
-import org.junit.jupiter.api.BeforeAll;
-
 import java.io.IOException;
 import java.nio.file.Files;
 import java.util.Arrays;
 import java.util.List;
 import java.util.Objects;
+import lombok.Getter;
+import lombok.SneakyThrows;
+import org.apache.spark.sql.Row;
+import org.apache.spark.sql.SparkSession;
+import org.junit.jupiter.api.BeforeAll;
 
 public class BaseSparkTest {
-    // todo: parameterize such that catalogs can be specified per cloud provider if desired
-    private static final String ServerUrl = System.getenv().getOrDefault("CATALOG_URI", "http://localhost:8080");
```

Comment:
Move those env variables into the `TestUtils` class.

Reply:
then move the TODO as well.